pages/index.jsx

```diff
@@ -149,8 +191,53 @@ const ContactButton = styled.button`
 `;
 
 const IssuesPage = () => {
+  const [modalOn, setModalOn] = useState(false);
+
   return (
     <div>
+      {modalOn ? (
+        <ModalContainer>
```

Comment:
This is a great use-case for `React.createPortal`. It's used to render an element outside the current component tree e.g. directly into the document.body or any other element.